examples/find_all_posts_mentioning_github.rb

```diff
@@ -21,7 +21,7 @@ def initialize(keyword, language: nil)
 
   # Run block for each blog post returned from api.
   # Uses a sliding time-based window to get all results.
-  # @see https://developer.twingly.com/resources/search/#pagination
+  # @see https://app.twingly.com/blog_search?tab=documentation
```

Comment:
```suggestion
  # @see https://app.twingly.com/blog_search?tab=documentation#pagination
```

Reply:
The reason is that we have talked about restructuring the documentation for the blog APIs to look more like the documentation for the other APIs. We might break the links then, which would make these out of date again. I just wanted to avoid having to update the links here yet another time 😅 

For example, something we've mentioned is moving out the "Sources" documentation to a separate tab, which shows for both Blog Search and LiveFeed, as it's the same for both APIs. The "Blogrank" and "Authority" sections are the same too, so maybe we should move those to separate tabs too. If we do that their URLs will change.

Reply:
Ah, right! Once the documentation has been updated I think it could be nice to add the new links here though :)